### DISCUSSION TITLE
drag and drop cell values

### DISCUSSION THREAD

**Discussion by jankrnavek** [vitmalina/w2ui#2519] [Q&A]
Hi,
is it possible to implement drag and drop cell values functionality on the grid ?
H.

**Comment by vitmalina** [comment]
All you can do by default is in this demo: https://w2ui.com/web/demos/#/combo/3. You can copy/paste into this grid. 

You can also do reorder rows: https://w2ui.com/web/demos/#/grid/26

There is no reorder cells

**Reply by jankrnavek** [reply]
Thanks a lot for your reply.
I need to set the grid in selectType: 'Row' mode, because of my client, and I need to have some option to quickly copy values between cells.
So it seems there is no this option
